maintenance/passport-azure-ad/lib/jsonWebToken.js

```diff
@@ -53,176 +36,200 @@ var hasCommonElem = (array1, array2) => {
  */
 exports.verify = function(jwtString, PEMKey, options, callback) {
 
-  /*********************************************************************
+  /**
```

Comment:
Formatting for these breaks is incorrect

Reply:
Incorrect formatting removed on #3996